pkg/yurtctl/cmd/yurtinit/options.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2020 The OpenYurt Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package yurtinit
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/pkg/errors"
+
+	"github.com/openyurtio/openyurt/pkg/yurtctl/constants"
+)
+
+// InitOptions  defines all the init options exposed via flags by yurtctl init.
+type InitOptions struct {
+	AdvertiseAddress        string
+	YurttunnelServerAddress string
+	ServiceSubnet           string
+	PodSubnet               string
+	Password                string
+	ImageRepository         string
+	OpenYurtVersion         string
+}
+
+func NewInitOptions() *InitOptions {
+	return &InitOptions{
+		ImageRepository: constants.DefaultOpenYurtImageRegistry,
+		OpenYurtVersion: constants.DefaultOpenYurtVersion,
+	}
+}
+
+func (o *InitOptions) Validate() error {
+	if err := validateServerAddress(o.AdvertiseAddress); err != nil {
+		return err
+	}
+	if o.YurttunnelServerAddress != "" {
+		if err := validateServerAddress(o.YurttunnelServerAddress); err != nil {
+			return err
+		}
+	}
+	if o.Password == "" {
```

Comment:
if run `yurtctl init` on master node,  i think `password` should be not needed.

Reply:
In the Linux system, the user login password is stored in /etc/shadow, the password is encrypted, and the encryption method may be different, such as sha512, md5, and password decryption requires tools. So i keep password here.
Is there a better way?

Reply:
@Peeknut I think that `sealer` should support install `OpenYurt controlplane` without password.

Reply:
The current version(0.6.0) of sealer does not support it. How about keeping this parameter for now? After they support, I will remove `password`.

Reply:
ok, i got it.